src/util/misc.c

```diff
@@ -9,6 +9,8 @@
 #include <ctype.h>
 #include <string.h>
 
+#define NON_ALPHABETIC_CHARS "0123456789!@#$%^&*()_+-=[]{}\\|;:'\",.<>/?`~§± "
```

Comment:
How did we selected the characters in this list? Why do we need to include `§ (167)` and `± (177)` but not `© (169)`?